package.json

```diff
@@ -197,6 +197,7 @@
 		"jest": "29.5.0",
 		"jest-jasmine2": "29.5.0",
 		"jest-junit": "13.0.0",
+		"jest-matchmedia-mock": "1.1.0",
```

Comment:
Should it be a dev dependency of the compose package though? Is there a good reason why it's required by the entire project?

Reply:
Probably, but the sub-packages don't have `devDependencies` specified, so I put it here. In any case, we'll go with custom mocking for the tests - https://github.com/WordPress/gutenberg/pull/48973#issuecomment-1463662327.

Reply:
I agree with Jarda on using custom mockups for it 👍 Seems like a sensible and flexible enough solution for the time being.

Reply:
> Should it be a dev dependency of the compose package though?

I think that a few years ago, we had serious issues with `devDependencies` being declared on individual packages. Something with Lerna, or hoisting, or was it NPM unable to find a binary for such a package, in the top-level or package-specific `node_modules/.bin` directory? I don't remember, maybe @blowery or @sirreal does.

In any case, Gutenberg seems to follow a convention where all `devDependencies` are declared at the top. While Calypso prefers that individual packages have their own `devDependencies`. Many of the were added during the Yarn migration though.

Reply:
Sorry, I don't remember the details about these conventions.